src/common/config.h

```diff
@@ -25,6 +25,7 @@
 #define OSD_REP_SPLAY   1
 #define OSD_REP_CHAIN   2
 
+#define VAL1 1
```

Comment:
> please be specific. in the case of your PR of https://github.com/ceph/ceph/pull/16447, replacing 1 with VAL1 does not help at all. if 1 is a magic number, for example, which is a sentry of a number sequence. we need to have a better name for it. but in this case, it's not.
> 

@tchaikov I think `#define ONE 1` would be good, Do you have any other meaningful name that is appropriate to the module.